examples/test_atari.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2017 Baidu Inc. All Rights Reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "games/arcade/arcade_simulator.h"
+
+using namespace simulator::arcade_game;
+using namespace simulator;
+
+DECLARE_int32(context);
+
+int main(int argc, char** argv) {
+    gflags::ParseCommandLineFlags(&argc, &argv, true);
```

Comment:
need to add these codes before this line:
#ifndef GFLAGS_GFLAGS_H_
namespace gflags = google;
#endif

same for other cpp files

Reply:
This is not easier. We need to add these lines for every file.